main.go

```diff
@@ -75,6 +79,20 @@ type gitLabAPI struct {
 	} `json:"namespace"`
 }
 
+// Struct for GitHub API
+type gitHubAPI struct {
```

Comment:
You use githubURL etc. everywhere, but here you use gitHubAPI. Small inconsistency.

Reply:
Noted. And I removed the GitHub API struct since we aren't currently using it.